pyspecdata/core.py

```diff
@@ -3544,22 +3543,23 @@ def cov_mat(self, along_dim):
         Parameters
         ==========
         along_dim:  str
-            the "observations" dimension of the data set (as opposed to
-            the variable)
+            the "variables" dimension of the data set (as opposed to
+            the variable). Note when rowvar = True for np.cov the rows are the
+            variables.
```

Comment:
I disagree with this change. You should specify the dimension that represents repeated measurements

Reply:
The reason why that makes sense is that that is the same dimension along which you would take the standard deviation, and the square of the standard deviation should match the diagonal of the covariance matrix.

In fact, in your example, you should explicitly show that that is true. That's where you're getting caught up.

Reply:
please look at the documentation for numpy cov. the rows (e.g. nScans) are termed the variables and the columns (e.g. t2) are the observations. I think it makes more sense to keep the phrasing of the function we are using

Reply:
https://numpy.org/doc/2.1/reference/generated/numpy.cov.html